src/clients/python/experimental_api_v2/examples/simple_http_v2_infer_client.py

```diff
@@ -80,20 +80,19 @@
 
     print(results.get_response())
 
-    # FIXME: Uncomment when DLIS 1162 is fixed.
-    # Get the output arrays from the
-    # output0_data = results.as_numpy('OUTPUT0')
-    # output1_data = results.as_numpy('OUTPUT1')
+    # Get the output arrays from the results
```

Comment:
Can you add a test which uses a wrong model_name to send out inference?

Reply:
Sure